util-flywayinstaller/src/main/java/com/github/cafapi/util/flywayinstaller/Migrator.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2022 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.cafapi.util.flywayinstaller;
+
+import com.github.cafapi.util.flywayinstaller.exceptions.InvalidConnectionStringException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.flywaydb.core.Flyway;
+import org.postgresql.ds.PGSimpleDataSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public final class Migrator
+{
+    private static final Logger LOGGER = LoggerFactory.getLogger(Migrator.class);
+    private static final String CONNECTION_URL_REGEX = "^.*\\/\\/.+:\\d+\\/";
+    private static final String CREATE_DATABASE = "CREATE DATABASE \"%s\"";
+    private static final String DOES_DATABASE_EXIST
+        = "SELECT EXISTS ( SELECT datname FROM pg_catalog.pg_database WHERE lower( datname ) = lower( ? ) );";
```

Comment:
You're only checking row existence so you don't need to output the column.
```suggestion
        = "SELECT EXISTS (SELECT NULL FROM pg_catalog.pg_database WHERE lower( datname ) = lower( ? ));";
```

Reply:
Done